_events/sr2021/kickstart.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Kickstart
+date: 2019-11-21 10:00:00
```

Comment:
Can we leave the time as undefined for now? It's not quite something we've defined yet.

Also, wrong year.

```suggestion
date: 2020-11-21
```